tests/uri.cpp

```diff
@@ -38,14 +38,6 @@ BOOST_AUTO_TEST_CASE(host_port)
     BOOST_CHECK_EQUAL( hostPort.getScheme(), "tcp" );
 }
 
-BOOST_AUTO_TEST_CASE(custom_schema_not_supported_yet)
```

Comment:
The schema is left untouched, so actually only line 46 changes. The testname is then custom_schema. For the zeq::URI this is supported then, but not yet for the publisher/subscriber. Will do later then when needed. inproc could be interesting to have 'local' GUI only for instance.